war/src/main/less/modules/side-panel-tasks.less

```diff
@@ -76,6 +76,8 @@
     margin: 0;
     width: 20px;
     height: 20px;
+    max-width: 20px;
+    max-height: 20px;
```

Comment:
If we move this to the block below, we _should_ be good to go, based on a quick look.

Guess, we could even go with 24px, to simulate the size of a medium-sized icon.